src/asciidoc/docs/publish/plugins.adoc

```diff
@@ -0,0 +1,45 @@
+= Using Koji plugins
+:page-slug: plugins
+:page-description: Plugins
+
+A plugin is software that can be added to a Koji after publication to provide additional functionality.
+Basically, it allows for the injection of dynamic code.
+Plugins can only be added by the creator of a Koji.
+They will not be passed down to remixes.
+
+Generally, a creator customizes a plugin for a particular Koji by specifying values for one or more fields.
+For example, the *Custom Boilerplate* plugin adds a popup requiring a user to acknowledge a licensing agreement before interacting with the Koji.
+The creator specifies the body text and the question text for the popup.
+
+== Advantages of plugins
+
+In addition to allowing the developer to easily add functionality to a Koji, plugins offer a few other advantages.
```

Comment:
```suggestion
In addition to allowing the creator to easily add functionality to a Koji, plugins offer a few other advantages.
```
Should this be the creator, since that's who adds it?